src/lib/Libattr/master_svr_attr_def.xml

```diff
@@ -1656,6 +1656,23 @@
 	<ECL>NULL_VERIFY_VALUE_FUNC</ECL>
 	</member_verify_function>
    </attributes>
+   <attributes>	
+   /* SRV_ATR_sync_mom_hookfiles_timeout */
+	<member_name><both>ATTR_sync_mom_hookfiles_timeout</both></member_name> <!-- "sync_mom_hookfiles_timeout" -->
+	<member_at_decode>decode_l</member_at_decode>
+	<member_at_encode>encode_l</member_at_encode>
+	<member_at_set>set_l</member_at_set>
+	<member_at_comp>comp_l</member_at_comp>
+	<member_at_free>free_null</member_at_free>
+	<member_at_action>NULL_FUNC</member_at_action>
+	<member_at_flags><both>MGR_ONLY_SET</both></member_at_flags>
+	<member_at_type><both>ATR_TYPE_LONG</both></member_at_type>
+	<member_at_parent>PARENT_TYPE_SERVER</member_at_parent>
+	<member_verify_function>
+	<ECL>verify_datatype_long</ECL>
+	<ECL>verify_value_non_zero_positive</ECL>
```

Comment:
I would expect a value of zero to disable the sync timeout. Is that not the case?

Reply:
I'll have to let @bayucan answer this (it is his code), but my assumption is that if the timeout is set, it will be used.  If you want to disable the timeout, you unset it.

Reply:
That is the correct, Bhroam.